src/modules/ie-objects/controllers/ie-objects.controller.ts

```diff
@@ -90,6 +90,11 @@ export class IeObjectsController {
 		);
 	}
 
+	@Get('newspaper-titles')
+	public async getNewspaperTitles() {
+		return this.ieObjectsService.getNewspaperTitles();
+	}
```

Comment:
for documentation purposes, also specify the response type here
```suggestion
	@Get('newspaper-titles')
	public async getNewspaperTitles(): Promise<NewspaperTitle[]> {
		return this.ieObjectsService.getNewspaperTitles();
	}
```